src/retest/cvtpcre.c

```diff
@@ -625,6 +638,12 @@ int main(int argc, char **argv)
 								fputc('~', out);
 								for (i = 0; i < regexp.len; i++) {
 									switch (regexp.s[i]) {
+									case '\\':
+										/* have to escape all backslashes in the regexp */
+										fputc('\\', out);
+										fputc('\\', out);
+										break;
+
```

Comment:
surprised we got away without this!

Reply:
Yeah.  The /x handling may have made this relevant.  I don't think it matters much for single-line regexps.  At least not in `testinput1`